src/graphql/schema.ts

```diff
@@ -30,6 +30,17 @@ export const typeDefs = `#graphql
     name: String!
     email: String!
     birthDate: String!
+    addresses: [Address]
```

Comment:
```suggestion
    addresses: [Address]!
```

Talvez seja melhor deixar obrigatório pq vc sempre tem a opção de retornar array vazio caso o usuário não tenha endereços. E aí não deixa uma possível redundância entre null e array vazio.